docs/TUMOR_ONLY_SNV_BENCH_RESULTS.md

```diff
@@ -0,0 +1,109 @@
+ - Variant `FILTER` = `PASS` or `INFO HotSpotAllele=1`
+ - `FILTER` GNOMAD_AF < 0.001 applied
+ - GDC Panel of Normals (PON) used - this is different from somatic workflow
+
+### Cell Line Tumor Only Benchmark, High + Medium Confidence Calls
+| Sample ID | True Positive | False Positive | False Negative | F1 Score | Precision | Sensitivity |
+|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
+| FD_T_2|  36812 |  31817 |  78222 |  0.4 | 0.54 | 0.32 |
+| FD_T_3|  37045 |  31991 |  78371 |  0.4 | 0.54 | 0.32 |
+| NV_T_2|  38596 |  39479 |  79858 |  0.39 | 0.49 | 0.33 |
+| NV_T_3|  38780 |  37809 |  79629 |  0.4 | 0.51 | 0.33 |
+| IL_T_1|  37817 |  36644 |  78340 |  0.4 | 0.51 | 0.33 |
+| IL_T_2|  37445 |  34935 |  77361 |  0.4 | 0.52 | 0.33 |
+| NS_T_1|  37504 |  40680 |  76893 |  0.39 | 0.48 | 0.33 |
+| NS_T_2|  35418 |  31957 |  75577 |  0.4 | 0.53 | 0.32 |
+### Synthetic Tumor Only Benchmark, High + Medium Confidence Calls
+| Sample ID | True Positive | False Positive | False Negative | F1 Score | Precision | Sensitivity |
+|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
+| syn_WHS_FD_N_2|  87309 |  48421 |  27757 |  0.7 | 0.64 | 0.76 |
+| syn_WGS_FD_N_3|  86609 |  48848 |  28215 |  0.69 | 0.64 | 0.75 |
+| syn_WGS_NV_N_2|  90573 |  59350 |  24607 |  0.68 | 0.6 | 0.79 |
+| syn_WGS_NV_N_3|  89412 |  60020 |  25588 |  0.68 | 0.6 | 0.78 |
+| syn_WGS_IL_N_1|  88403 |  58522 |  25918 |  0.68 | 0.6 | 0.77 |
+| syn_WGS_IL_N_2|  87446 |  56718 |  26852 |  0.68 | 0.61 | 0.77 |
+| syn_WGS_NS_N_1|  86835 |  75729 |  27403 |  0.63 | 0.53 | 0.76 |
+| syn_WGS_NS_N_2|  84797 |  55039 |  29212 |  0.67 | 0.61 | 0.74 |
+
+## Variant Filter fine-tuning
+Given that the focus is on reducing FPs from tumor only calling and that all synthetic and cell line samples have very high overlap in calls, only one samples was used to benchmark various filtering strategies and combinations.
+Many combinations of filters were performed from the two publications.
+Below we attempt to concisely summarize the story and results.  
+
+### Test synthetic sample `syn_WGS_IL_N_1` HIVE filter components and final combination
+- RAW: the protected file as output by our pipeline
+- PASS: Stripped the public file soft filters from the RAW and grabbed only the PASS records
+- PASS+GNO 0.1%: Applied the KF gnomAD filter (exclude gnomad >= 0.001) directly to the PASS file. These results should be a pretty close approximation of what our public files look like
+- PASS+GNO 0.003%: Applied the HIVE gnomAD filter (exclude gnomad >= 0.00003) directly to the PASS file. I used this as a comparison to see just how much value we were getting from the other filtration steps
+- PASS+PON: Removed any records from the PASS file that were also found in the GATK PON
+- PASS+PON+dbSNP: Removed any records from PASS+PON that were labeled COMMON in dbSNP
+- PASS+PON+dbSNP+ClinVar: Removed any records from PASS+PON+dbSNP that had "benign" in their worst VEP assigned CLN_SIG
```

Comment:
This seems unclear to me what it means reading as someone who has not used this pipeline before: 
"had "benign" in their worst VEP assigned CLN_SIG"

Reply:
It's VEP jargon. `CLN_SIG` is the VEP key for "ClinVar clinical significance of the dbSNP variant". See [Default VEP output](https://useast.ensembl.org/info/docs/tools/vep/vep_formats.html). ClinVar significance is just an annotation about the significance of any given variant: https://www.ncbi.nlm.nih.gov/clinvar/docs/clinsig/#clinsig_scv. 

VEP can assign many different consequences to the variants based on the alternative alleles. The concept of "worst" comes into play when you have a lot of different annotations for a given variant. In that case, you often want to check what's the "worst" case scenario for that variant. 

So what this sentence is saying is: for any given variant, look through all the annotations and see what the worst ClinVar significance is. If its "only" benign, then we drop the variant.